salvo/src/lib/run_benchmark.py

```diff
@@ -116,11 +194,10 @@ def _pull_or_build_nh_benchmark_image(self, \
        Args:
         images: the DockerImages appearing in the control object
     """
-
+    di = docker_image.DockerImage()
```

Comment:
We've changed `di` in previous PRs to use a more descriptive name.  The renamed variable is `image_manager` IIRC.  Can we do that here?